src/components/Ticket/components/TwoLineHeading.styled.ts

```diff
@@ -0,0 +1,15 @@
+import styled from '@emotion/styled';
+import { Stack } from '@mui/material';
+
+export const TwoLineHeadingWrapper = styled(Stack)(({ theme }) => ({
+    h3: {
```

Comment:
h1 and h2 is there?

Reply:
This component is used inside the Ticket component. Ticket already has a h1 component. But h2 was still missing. So changed h3 and h4 to h2 and h3
Resolved in a7ece662e28ede42e328db75978d2d9d9560689c